python/Ganga/GPIDev/Base/VPrinter.py

```diff
@@ -120,7 +124,10 @@ def simpleAttribute(self, node, name, value, sequence):
             # else:
             #    print 'no transformation'
             if isclass(value):
-                print(self.indent(), name, '=', repr(value), end = '', file=self.out)
+                if self._interactive is True:
+                    print(self.indent(), name, '=', str(value), end = '', file=self.out)
+                else:
+                    print(self.indent(), name, '=', repr(value), end = '', file=self.out)
```

Comment:
As far as I can tell, this is reason for all those `interactive` flags being passed around that you've added. What is the reason for the difference and what does 'interactive' mean in this context. This would at least need a docstring explaining it but this change overall should probably have been in a separate PR so we could discuss it separately.